src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
 /**
  * @param {number} days
  *
+ *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRASE_OF_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let basePrises = days * PRASE_OF_DAY;
```

Comment:
The variable `basePrises` has a typo and should be named `basePrice` to accurately reflect its purpose. Also, consider using `const` instead of `let` since `basePrice` is not reassigned after its initial calculation, which adheres to checklist item 3.